tests/cluster.go

```diff
@@ -135,6 +135,7 @@ func (s *TestServer) Destroy() error {
 func (s *TestServer) ResignLeader() error {
 	s.Lock()
 	defer s.Unlock()
+	s.server.GetMember().ResetLeader()
```

Comment:
Why does it need to be explicitly executed here?

Reply:
It is indeed a bit redundant, but I want to clearly express the leader’s resign behavior.